<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="interface_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Agence</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>id Produit</string>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>80</y>
        <width>20</width>
        <height>231</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Langage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Francais</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Anglais</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Arabe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>0</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Liste des contrats :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cin_Contrat">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idAgence_Contrat">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>350</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_idAgence">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>340</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>id produit</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_date">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>360</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_type">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>380</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>310</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>id produit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idProduit_Contrat">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_typeContrat">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>300</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifierContrats">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimerContrat">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>530</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouterContrat">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>350</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>date contrat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>420</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>id contrat pour modifier ou supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idsuppmodifContrat">
      <property name="geometry">
       <rect>
        <x>462</x>
        <y>440</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewContrat">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>80</y>
        <width>321</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_Contrat">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>352</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
